contrib/core/actions/uuid.yaml

```diff
@@ -0,0 +1,6 @@
+---
+name: uuid
```

Comment:
Yeah, "type" may not be the most ideal name for that param. It will for sure clash with the "type" field that st2 exposes to declare "data type" for parameters. I'd call it uuid_type and it's a good idea. +1 to that.